environment.yml

```diff
@@ -23,7 +23,7 @@ dependencies:
 - traitsui
 - imageio
 - tqdm
-- spyder-kernels
+- spyder-kernels>=1.9.2
```

Comment:
This should be `>=1.10.0`, if you plan to use Spyder 4.2.1

Reply:
thanks for the heads-up @ccordoba12 !

Reply:
```suggestion
- spyder-kernels>=1.10.0
```